interfaces/IBF-dashboard/src/app/pages/dashboard/status-report/status-report.page.html

```diff
@@ -0,0 +1,37 @@
+<ion-header class="ion-no-border">
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-button title="Home">
+        <ion-icon name="apps" slot="icon-only"></ion-icon>
```

Comment:
Added now, but to be fair in homepage this menu actually has a different meaning (open dev-menu if admin, otherwise nothing). So not totally consistent.